### DISCUSSION TITLE
New to Git/new to HomeBrew having trouble installing PIA(Private internet access)

### DISCUSSION THREAD

**Discussion by Phototos** [Homebrew/discussions#674] [Everyday usage] [answered]
I just did a fresh install of OS 11.2 Big sur/MacBookPro 2015(intel)
Trying out homebrew for the first time
installed a couple applications successfully with homebrew

When I got to PIA it said this...

Downloading https://installers.privateinternetaccess.com/download/pia-macos-######################################################################## 100.0%
==> Installing Cask private-internet-access
To complete the installation of Cask private-internet-access, you must also
run the installer at:
  /usr/local/Caskroom/private-internet-access/2.6.1-05824/Private Internet Access Installer.app
🍺  private-internet-access was successfully installed!

Not sure how to complete the installation. 

I haven't used Terminal much, but I had a commodore 64 when i was a kid so i get the concept of command-lines. Figure i need to enter the right command-line, to finish the install but I don't know where to learn them yet.

Thanks in advance for any advice.

**Comment by carlocab** [comment, marked as answer]
Try
```
open /usr/local/Caskroom/private-internet-access/2.6.1-05824
```
This will open a Finder window to that directory. You can then run the `Private Internet Access Installer.app` from there.

**Reply by Phototos** [reply]
Thank you!
